internal/statestorage/redis/redispb/matchobject.go

```diff
@@ -129,7 +131,7 @@ func Watcher(ctx context.Context, pool *redis.Pool, pb om_messages.MatchObject)
 				results = om_messages.MatchObject{Id: pb.Id}
 				err = UnmarshalFromRedis(ctx, pool, &results)
 				if err != nil {
-					moLog.Debug("No new results")
+					moLog.Debug("No new results ", err)
```

Comment:
Would this be Debugf("... %s")?

Reply:
I'll keep that in mind. I know there's a bunch of log format improvements to chase down.

Reply:
Fixed :)